starter-code/MyLinkedList.java

```diff
@@ -12,51 +9,111 @@ public int getSize() {
 	}
 
 	public boolean remove(int index) {
-		//to-do
+        if(index < 0 || index > size) {
+            return false;
+        }
+        Node<T> current = head;
+
+        if(head != null) {
+            while(index > 0) {
+                if(current.getNext() == null) {
+                    return false;
+                }
+                current = current.getNext();
+                index--;
+            }
+            current.setNext(current.getNext().getNext());
```

Comment:
What if `index` is zero, i.e. the caller wants to remove the head? I don't think your method will do that